tools/StaticAnalysis/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace StaticAnalysis.DependencyAnalyzer
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            if (args == null || args.Length < 1)
+            {
+                throw new InvalidOperationException("Please pass a valid directory name as the first parameter");
+            }
+
+            var installDir = args[0];
+            if (!Directory.Exists(installDir))
+            {
+                throw new InvalidOperationException("You must pass a valid directory as the first parameter");
+            }
+
+            var directories = new List<string>
+            {
+                Path.Combine(installDir, @"ResourceManager\AzureResourceManager\"),
+                Path.Combine(installDir, @"ServiceManagement\Azure\")
+            };
+
+            var reportsDirectory = Directory.GetCurrentDirectory();
+            if (args.Length > 1 && Directory.Exists(args[1]))
+            {
+                reportsDirectory = args[1];
```

Comment:
Please add warning message that will indicate parameter 2 is not a valid folder and the output will be done in the current folder (and print that out so we would not wonder where was the current folder on the CI machine)